### ISSUE TITLE
Terrain texture paint wasn't fixed.

### ISSUE THREAD

**Issue by Hamada880** [TheGameCreators/GameGuruRepo#2967]
Changing the texture to DDs darken the images to black. Also crashes the software during the launching. Xconvert used to convert textures to dds and also Photoshop plug-in but the result is same black textures. Is there way to explain how to fix this issue?

**Comment by GraPhiX-Guru** [comment]
@Hamada880 xTextureConverter is the tool for MAX terrains xConvert will convert textures but the textures for MAX are specific.
https://youtu.be/vLGWz_2sqT0
hope this helps

**Comment by Hamada880** [comment]
Will tested it out, much appreciated it.

**Comment by Necrym59** [comment]
My question is why such a convoluted way of doing this with the textures, if its going to be for users to use the own textures a converter needs to be in place to create the correct group of dds files or users will have to buy a product just to use it, it just seems strange to me.

**Comment by PaulSJ** [comment]
Its a trade off between quality and memory usage. BC7 is the best quality for color images, BC5 is the best for normal maps, and the surface is less noticeable so gets BC1 for best compression. Also all the textures must be 2048x2048 due to how they are used internally.

The tool I attached above can be used if you want to use your own textures. It isn't officially supported because of this complexity and trade offs, but it's there if you want to use it.

**Comment by Necrym59** [comment]
Hi Paul I understand the tecnicalities, im just interested as to how its going to be implemented for the common user in MAX all they want to do is add 'texture' will it be an automated conversion?

**Comment by PaulSJ** [comment]
Yes, it is possible we could add this process to the editor to make it easier. But given the amount of work it takes to make a good looking terrain texture (it needs matching color, normal, roughness, and AO maps for the PBR renderer), it's not something that will be commonly used, so it's not a high priority at the moment.

**Comment by Necrym59** [comment]
Yeah, my thought was it could generate those additional maps from the one being added. Personally i can get by doing all that stuff i was just thinking of other end users.
